etc/calibre.profile

```diff
@@ -22,7 +22,7 @@ nonewprivs
 noroot
 nosound
 novideo
-protocol unix,inet,inet6
+protocol unix,inet,inet6,netlink
```

Comment:
Is the message about netlink purely cosmetic or does it actually break calibre from working properly?